webapp/.env.example

```
@@ -1,23 +1,17 @@
 # copy to .env.local for development
-# Application URL
-WEBAPP_URL="https://marumie.team-mir.ai"
 
-# Cache refresh
-DATA_REFRESH_TOKEN="your-secret-refresh-token"
+# === 必須 ===
+DATABASE_URL="postgresql://postgres:postgres@127.0.0.1:54332/postgres"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)